packages/vaadin-template-renderer/src/vaadin-template-renderer.js

```diff
@@ -1,19 +1,70 @@
 import { FlattenedNodesObserver } from '@polymer/polymer/lib/utils/flattened-nodes-observer.js';
 
+import { GridElement } from '@vaadin/vaadin-grid/src/vaadin-grid.js';
+import { GridColumnElement } from '@vaadin/vaadin-grid/src/vaadin-grid-column.js';
+import { GridColumnGroupElement } from '@vaadin/vaadin-grid/src/vaadin-grid-column-group.js';
+
 import './vaadin-template-renderer-templatizer.js';
+import './vaadin-template-renderer-grid-templatizer.js';
+
 import { Templatizer } from './vaadin-template-renderer-templatizer.js';
+import { GridTemplatizer } from './vaadin-template-renderer-grid-templatizer.js';
 
-function createRenderer(template) {
-  const templatizer = Templatizer.create(template);
+function createRenderer(component, template, TemplatizerClass = Templatizer) {
+  const templatizer = TemplatizerClass.create(component, template);
 
-  return (root, _owner, model) => {
-    template.__templatizer = templatizer;
-    template.__templatizer.render(root, model);
+  const renderer = (root, _owner, model) => {
+    templatizer.render(root, model);
   };
+
+  template.__templatizer = templatizer;
+  renderer.__templatizer = templatizer;
```

Comment:
Looks like this reference isn't used anywhere

Reply:
Removed. Although, probably this reference will be necessary again for https://github.com/vaadin/web-components/pull/2052#discussion_r647226521. I'll introduce it there.